torchvision/csrc/io/video_reader/video_reader.cpp

```diff
@@ -653,10 +655,12 @@ torch::List<torch::Tensor> read_video_from_file(
 }
 
 torch::List<torch::Tensor> probe_video_from_memory(torch::Tensor input_video) {
+  C10_LOG_API_USAGE_ONCE("torchvision.io.probe_video_from_memory");
```

Comment:
@prabhat00155 Do we want to log this? Is this considered part of public API?

Reply:
Yeah, see this: https://github.com/pytorch/vision/blob/01ffb3ae9f9cc3a445bcaabf787251a05e67c7a7/torchvision/csrc/io/video_reader/video_reader.cpp#L667